exercises/minesweeper/minesweeper_test.py

```diff
@@ -1,162 +1,71 @@
-""" Tests for the minesweeper exercise
-
-Implementation note:
-The board function must validate its input and raise a
-ValueError with a *meaningful* error message if the
-input turns out to be malformed.
-"""
```

Comment:
Implementation notes should go in `.meta/hints.md` (create it if it doesn't already exist). When `hints.md` is created/modified, the exercise README must be [regenerated](https://github.com/exercism/python/blob/master/CONTRIBUTING.md#generating-exercise-readmes)